.github/workflows/dotnet-instr-deployer-add-on.yml

```diff
@@ -4,13 +4,27 @@ on:
   push:
     paths:
       - 'packaging/dotnet-instr-deployer-add-on/**'
+      - '.github/workflows/dotnet-instr-deployer-add-on.yml'
   pull_request:
     paths:
       - 'packaging/dotnet-instr-deployer-add-on/**'
+      - '.github/workflows/dotnet-instr-deployer-add-on.yml'
   workflow_dispatch:
+    inputs:
+      splunk_uf_version:
+        description: 'Splunk UF version'
+        required: false
+        default: '9.4.0'
+      splunk_uf_build_hash:
+        description: 'Splunk UF build hash'
+        required: false
+        default: '6b4ebe426ca6'
 
 env:
   GO_VERSION: 1.23.6
+  # Provide default values for non-manually triggered executions
+  splunk_uf_version: ${{ github.event.inputs.splunk_uf_version || '9.4.0' }}
```

Comment:
Notice that this doesn't need to be in sync with the defaults for `workflow_dispatch`, it is nice if they are but not needed.